yscope-k8s/templates/object-store/minio.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: "v1"
```

Comment:
We don't need this at the moment since we are launching log-viewer via the package.